crates/ruff/Cargo.toml

```diff
@@ -12,8 +12,6 @@ license = "MIT"
 
 [lib]
 name = "ruff"
-crate-type = ["cdylib", "rlib"]
-doctest = false
```

Comment:
We want doctests and `crate-type` is no longer necessary now that the wasm code lives in `ruff_wasm`